internal/controller/cluster_monitoring_controller.go

```diff
@@ -80,25 +81,29 @@ func (r *ClusterMonitoringReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		return ctrl.Result{}, errors.WithStack(err)
 	}
 
+	ctx = context.WithValue(ctx, "cluster", cluster.Name)                  // nolint
```

Comment:
Can you explain why the nolint?

Reply:
I'm gonna add a comment

Reply:
Done

Reply:
Can you try:

```go
logger := log.FromContext(ctx)
                       .WithValues("cluster", cluster.Name)
                       .WithValues("installation", r.ManagementCluster.Name)
ctx = log.IntoContext(ctx, logger)
```

Reply:
I think we have a bit too many clusters but we can sort this out later :)